recipes/libxmlpp/2.x.x/conanfile.py

```diff
@@ -0,0 +1,137 @@
+from conans import ConanFile, Meson, tools
+from conan.tools.files import rename
+from conans.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import is_msvc
+import shutil
+import os
+
+
+class LibXMLPlusPlus(ConanFile):
+    # FIXME: naming the library "libxml++" causes conan not to add it to the
+    # environment path on windows
+    name = "libxmlpp"
+    homepage = "https://github.com/libxmlplusplus/libxmlplusplus"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "libxml++ (a.k.a. libxmlplusplus) provides a C++ interface to XML files"
+    topics = ["xml", "parser", "wrapper"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    generators = "pkg_config"
+    exports_sources = "patches/**"
+    short_paths = True
```

Comment:
```suggestion
```
Short package name. Does it really pass 256 path character?

Reply:
I don't think so. I thought it wouldn't hurt.

Reply:
We avoid `short_paths` always. It's only used when really needed, as it customize the conan data folder and may affect users (folder permission, etc ...)